src/simulations/action_translator.py

```diff
@@ -341,8 +370,27 @@ def actions_to_env_vars(self, loads, home_vars, action, date, time_step):
         # loads: flex_cons, tot_cons_loads
         # home_vars: netp, bool_flex, tot_cons
         # bool_penalty
+        # flexible_q_car
+
+        return loads, home_vars, bool_penalty, flexible_q_car
+
+    def _calculate_flexible_q_car(self, indiv_flexible_store_action, indiv_flexible_q_car_action):
+        if indiv_flexible_q_car_action > 0:
+            charge = indiv_flexible_store_action
```

Comment:
you could have `indiv_flexible_q_car_action > 0` and  `ds <0`? it could be a discharge?